tests/sentry/api/test_event_search.py

```diff
@@ -1198,329 +1143,186 @@ def test_aggregate_filter_and_normal_filter_in_condition_with_or(self):
-                                        [
-                                            "and",
-                                            [
-                                                ["or", _neq("username", "bar")],
-                                                [
-                                                    "and",
-                                                    [
-                                                        ["or", _neq("ip", "bar")],
-                                                        ["or", _neq("id", "bar")],
-                                                    ],
-                                                ],
-                                            ],
-                                        ],
-                                    ],
-                                ],
-                            ],
-                        ],
-                        [
-                            "or",
-                            [
-                                _eq("email", "foobar"),
-                                [
-                                    "or",
-                                    [
-                                        _eq("username", "foobar"),
-                                        ["or", [_eq("ip", "foobar"), _eq("id", "foobar")]],
-                                    ],
-                                ],
-                            ],
-                        ],
+                        _or(
+                            _or(
+                                _ueq("email", "foo"),
+                                _or(
+                                    _ueq("username", "foo"),
+                                    _or(_ueq("ip", "foo"), _ueq("id", "foo")),
+                                ),
+                            ),
+                            _and(
+                                _uneq("email", "bar"),
+                                _and(
+                                    _uneq("username", "bar"),
+                                    _and(_uneq("ip", "bar"), _uneq("id", "bar"),),
+                                ),
+                            ),
+                        ),
+                        "=",
+                        1,
+                    ],
+                    [
+                        _or(
+                            _ueq("email", "foobar"),
+                            _or(
+                                _ueq("username", "foobar"),
+                                _or(_ueq("ip", "foobar"), _ueq("id", "foobar")),
+                            ),
+                        ),
+                        "=",
+                        1,
                     ],
                 ],
             ),
         ]
 
         for test in tests:
             result = get_filter(test[0])
-            assert [[test[1], "=", 1]] == result.conditions, "cond: " + test[0]
+            print("EXP", test[1])
+            print("ACT", result.conditions)
```

Comment:
should this be here?

Reply:
I got a fever! And the only cure, is more `print` statements!